### PR TITLE
Add respawn cvar

### PR DESCRIPTION
**Pull request by element109**

### CODE REVIEW

**Comment by BenLubar** [comment]
I'm confused about this one - it's a vscript function, so it has the same requirements to call as `Convars.SetValue`. What does having a cvar to disable it add?

**Comment by element109** [comment]
I have a console command that calls this chain.

File: src/game/shared/swarm/asw_gamerules.cpp [hunk 1]
```diff
@@ -4048,9 +4035,19 @@ void CAlienSwarm::Resurrect( CASW_Marine_Resource * RESTRICT pMR )
...
+ConVar asw_respawn_marine_enable( "asw_respawn_marine_enable", "0", FCVAR_CHEAT, "Enables respawning marines.", true, 0, true, 1 );
+
+bool CAlienSwarm::ScriptResurrect( CASW_Marine_Resource* RESTRICT pMR, Vector vecSpawnPos )
 {
-	if ( !SpawnMarineAt( pMR, vecSpawnPos + Vector(0,0,1), QAngle(0,0,0 ), true ) )
+	if ( !pMR || GetGameState() != ASW_GS_INGAME ) return false;
+
+	if ( !asw_respawn_marine_enable.GetBool() )
+	{
+		Msg( "Respawning marines is not enabled on this server.\n" );
```
[the remark this answers is not in the document]
Comment (element109) [reply src/game/shared/swarm/asw_gamerules.cpp:4046]: That is a console message, they typically are not translated, but a translated hud message could be added.

[the remark this answers is not in the document]
Comment (Blueberryy) [reply src/game/shared/swarm/asw_gamerules.cpp:4046]: Ah okay, i thought that was hud message

**Pull request merged**